src/function/index.test.ts

```diff
@@ -221,6 +223,104 @@ describe('exponential function', () => {
         }).toThrow('Unknown color space: "invalid"');
     });
 
+    test('colorArray', () => {
```

Comment:
Not sure I understand from the test string what these tests are checking, is it possible to improve this?